src/officehours_api/models.py

```diff
@@ -94,6 +94,7 @@ class Queue(SafeDeleteModel):
         models.CharField(max_length=20, choices=get_backend_types(), blank=False),
         default=get_default_allowed_backends,
     )
+    physLocation = models.CharField(max_length=100, blank=True)
```

Comment:
I'm sorry, I still want this to be `phys_location` so it uses the same naming convention as the other fields. You will also need/want to use the underscore format in the FE with `api.ts` and in `models.ts` (and consequently with any uses of `Queue` in the components you changed). I'll try to comment in the right places to save you some time. Migration will also have to be re-run, and serializers updated.